src/ReleaseHistory.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* BUGFIX: Fix Error BA2006: '...' was compiled with one or more modules which were not built using minimum required tool versions [533](https://github.com/microsoft/binskim/pull/533)
```

Comment:
>* BUGFIX: Fix Error BA2006: '...' was compiled with one or more modules which were not built using minimum required tool versions  [](http://example.com/codeflow?start=0&length=130)

This seems a little too generic. Can we specify which instances this is being fixed for?